code/modules/mining/abandoned_crates.dm

```diff
@@ -43,8 +43,12 @@
 					if(sanitised[i] == sanitised[j])
 						sanitycheck = FALSE //if a digit is repeated, reject the input
 			if(input == code)
+				to_chat(user, span_notice("A green light flashes."))
```

Comment:
1. Why aren't we actually setting the abandoned crate's sprite to be the unlocked green?
2. If there is a reason to not do #1, make this a balloon alert instead (`balloon_alert(user, "a green light flashes")`)

Reply:
That's actually done in togglelock() code and I realized that now. Will remove it.